tests/test_session_1_byref_false.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
```

Comment:
filename is too long.  don't make it any longer than `test_session_1.py`

Reply:
Agreed.